src/error.ts

```diff
@@ -198,6 +198,112 @@ export class MongoRuntimeError extends MongoDriverError {
   }
 }
 
+/**
+ * An error generated when a batch command is reexecuted after one of the commands in the batch
+ * has failed
+ *
+ *
+ * @public
+ * @category Error
+ */
+export class MongoBatchReExecutionError extends MongoRuntimeError {
+  constructor(message: string) {
+    super(message);
+  }
+
+  get name(): string {
+    return 'MongoBatchReExecutionError';
+  }
+}
+
+/**
+ * An Error thrown when the user attempts to operate on a database or collection through a MongoClient
```

Comment:
```suggestion
 * An error thrown when the user attempts to operate on a database or collection through a MongoClient
```
Just for consistency.